tests/coverage_lcov/test_lcov.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+function outerdir(){
+   echo "$1" | sed -E 's@/[^.]+.*@/@'
+}
+DIR_BUILD=$PWD
+DIR_OUTER=$(outerdir "$0")
+if [ "$DIR_OUTER" = "./" ]
+then
+     echo "The script must be run from the 'facelift/build'"
+     exit 1
+fi
+DIR_SRC=$(cd "$DIR_OUTER"; pwd -P)
```

Comment:
DIR_SRC=$(cd $(dirname "$0")/..; pwd -P)

Reply:
then you can remove that outerdir stuff....

Reply:
We have already discussed the "related path .." and how to deduce the source path from the dirname.
If the this variant you proposed suits you, then this is not a problem.
Thank you.

Reply:
It is simple and it does work, so why would we need anything else ? Why did you change that in the first place ?

Reply:
Q: Why did you change that in the first place?
A: I did to conclusion from your short sentences that is required is more "smart" solution for non trivial decision with traverse of folders.
J'ai pris vos paroles trop au sérieux. Vous vouliez le plus simple, je voulais le meilleur. Ne vous fâchez pas, tout ira bien de toute façon.